django_pandas/managers.py

```diff
@@ -238,7 +238,7 @@ def to_timeseries(self, fieldnames=(), verbose=True,
         return df
 
     def to_dataframe(self, fieldnames=(), verbose=True, index=None,
-                     coerce_float=False):
+                     coerce_float=False, datetime_index=True):
```

Comment:
Is there a way you could set datetime_index to False by default in the to_dataframe method? That way the default index would be left alone.
~~~~python
df = CarBrands.objects.to_dataframe()
~~~~
I just think it would be convenient that this code should return an index like this [0,1,2,...,n].

If you want it to be a datetime column, this could be an example
~~~~python
df = CarSales.objects.to_dataframe(index='sale_dttm', datetime_index=True)
~~~~